ui/pages/bridge/types.ts

```diff
@@ -1,3 +1,25 @@
+import { BigNumber } from 'bignumber.js';
+
+export type L1GasFees = {
+  l1GasFeesInHexWei?: string; // l1 fees for approval and trade in hex wei, appended by controller
+};
+
+// Values derived from the quote response
+export type QuoteMetadata = {
+  totalNetworkFee: { raw: BigNumber; fiat: BigNumber | null }; // gasFees + relayerFees
+  toTokenAmount: { raw: BigNumber; fiat: BigNumber | null };
+  adjustedReturn: { fiat: BigNumber | null }; // destTokenAmount - totalNetworkFee
+  sentAmount: { raw: BigNumber; fiat: BigNumber | null }; // srcTokenAmount + metabridgeFee
+  swapRate: BigNumber; // destTokenAmount / sentAmount
+  cost: { fiat: BigNumber | null }; // sentAmount - adjustedReturn
```

Comment:
Mentioned this above, but would be good to add comment on what exactly fiat is being measured in

Reply:
Good call - added a comment